### PR TITLE
Adding index for tickets.  

### PR DESCRIPTION
**Pull request by dwickwire**
Drops the average ticket query from 400ms to 6ms.

```sql
federation=> CREATE INDEX cas_provider_serviceticket_index ON cas_provider_serviceticket USING btree (ticket);
CREATE INDEX
federation=> EXPLAIN (COSTS FALSE, FORMAT JSON)   SELECT "cas_provider_serviceticket"."id", "cas_provider_serviceticket"."ticket", "cas_provider_serviceticket"."created", "cas_provider_serviceticket"."user_id", "cas_provider_serviceticket"."service" FROM "cas_provider_serviceticket" WHERE "cas_provider_serviceticket"."ticket" = '<ticket>';
                                    QUERY PLAN                                     
-----------------------------------------------------------------------------------
 [                                                                                +
   {                                                                              +
     "Plan": {                                                                    +
       "Node Type": "Index Scan",                                                 +
       "Scan Direction": "Forward",                                               +
       "Index Name": "cas_provider_serviceticket_index",                          +
       "Relation Name": "cas_provider_serviceticket",                             +
       "Alias": "cas_provider_serviceticket",                                     +
       "Index Cond": "((ticket)::text = '<ticket>'::text)"+
     }                                                                            +
   }                                                                              +
 ]
(1 row)
```

### CODE REVIEW

File: cas_provider/migrations/0005_add_index_to_cas_provider_tickets.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        db.execute(u'DROP INDEX IF EXISTS cas_provider_serviceticket_index; CREATE INDEX cas_provider_serviceticket_index ON cas_provider_serviceticket USING btree (ticket);')
```
Comment (ryanmrubin) [inline cas_provider/migrations/0005_add_index_to_cas_provider_tickets.py]: Should you remove the IF EXISTS from these, too?

And is there a reason not to do the same as in federation, where you drop it on prod before deploying and not drop index from the forwards method?

**Comment by eykd** [comment]
Will this suffer from the same `IF EXISTS` problem encountered in https://github.com/Crossway/federation/pull/98? Seems like it should be unnecessary.

**Comment by dwickwire** [comment]
I thought I had removed it but clearly hadn't.  Need to each lunch – I'm loosing my focus. :)  All set.

**Comment by ryanmrubin** [comment]
:+1:

**Pull request merged**